### PR TITLE
PIN-4528: add datalake-data-export cronjob

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

**Review by micdes-pagopa — APPROVED** [review]

File: kubernetes/configs/test/versions.sh [hunk 1]
```diff
@@ -40,3 +40,4 @@ JOB_ANAC_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_IVASS_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_PN_CONSUMERS_IMAGE_VERSION="1.0.23"
 JOB_DTD_METRICS_IMAGE_VERSION="1.0.23"
+JOB_DATALAKE_DATA_EXPORT_IMAGE_VERSION="1.0.x-latest"
```
Comment (galales) [inline kubernetes/configs/test/versions.sh:43]: is it ok to have the latest here?

File: kubernetes/configs/prod/versions.sh [hunk 2]
```diff
@@ -40,3 +40,4 @@ JOB_ANAC_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_IVASS_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_PN_CONSUMERS_IMAGE_VERSION="1.0.23"
 JOB_DTD_METRICS_IMAGE_VERSION="1.0.23"
+JOB_DATALAKE_DATA_EXPORT_IMAGE_VERSION="1.0.x-latest"
```
Comment (galales) [inline kubernetes/configs/prod/versions.sh:43]: is it ok to have the latest here?

File: kubernetes/jobs/datalake-data-export/qa/cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app: {{SERVICE_NAME}}
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+spec:
+  schedule: "45 6 * * *"
+  suspend: true
```
Comment (galales) [inline kubernetes/jobs/datalake-data-export/qa/cronjob.yaml:10]: this is fine, just a note to us: if we plan to have differences in manifests across environments we should probably think about a strategy for automations

File: kubernetes/configs/test/versions.sh (hunk 1 above)
Comment (beetlecrunch) [reply kubernetes/configs/test/versions.sh:43]: Yes, it's just a placeholder waiting for the 1.0.24 version.

File: kubernetes/configs/prod/versions.sh (hunk 2 above)
Comment (beetlecrunch) [reply kubernetes/configs/prod/versions.sh:43]: same

**Review by galales — APPROVED** [review]

**Pull request merged**